cron/dump_clean_daily

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Performs a filtered dump of the measurements table.
+#
+# The filtering is the same as that used by the iOS app.
+#
+# This differs from dump_measurements, which does not filter data at all.
+# The schema returned is also different; redundant/unused columns have been eliminated, and id/user_id finally included.
+# The ORDER BY clause was removed for performance.
+
+set -euo pipefail
+
+source cron_env.sh
+
+cd ../public/system
+
+psql -f "${CRON_DIR}/dump_clean_daily.sql"
+
+tar -czf dump_clean_daily.tar.gz dump_clean_daily-out.csv
```

Comment:
I'm not sure this command is correct.

https://github.com/Safecast/safecastapi/pull/699/files#diff-e634e33c73747f8d7f1b8d4f53cc622aR56

The sql script places this file at `/tmp/dump_clean_daily.csv`

Reply:
Yep, you are right, it runs it in the same directory. I'll remove `/tmp`